### ISSUE TITLE
"Project+ Dolphin Launcher.elf" Attempts to Load Files that Do Not Exist in Linux

### ISSUE THREAD

**Issue by ejtejada** [Birdthulu/FPM-Installer#4]
Hello Burdthulu,

So the elf file from your script tries  to load some files that don't exist in the latest sd.raw that your script links to (or at least it doesn't on the linux build?).
Just a few examples:

`05:41:534 Core/HW/EXI/EXI_DeviceIPL.cpp:343 N[OSREPORT]: File Read  (SD): Open failed (/Project+/pf/sound/strm/Z03.brstm)
05:50:384 Core/HW/EXI/EXI_DeviceIPL.cpp:343 N[OSREPORT]: File Read  (SD): Open failed (/Project+/pf/fighter/wario/FitWario11.pac)`

I also don't know how or where to get the code to recompile this .elf to fix the issue.
I mounted the sd.raw using
`sudo mount -t vfat -o loop,offset=0 sd.raw ~\some\mount`
Just to see if these files exist.
/Project+/pf/sound/strm/ contains organized and named music files, so various music file loads fail at launch.
I looked in /Project+/pf/fighter/wario/
And saw that is FitWario11.pac unused, as a text files says:
"Slots 6-11: Use the Classic Wario boneset."

These issues don't cause problems at first, until I allow CPU's to fight for more than 2 minutes. At some point, something fails to load, and I get a new error:
`09:37:212 Core/HW/EXI/EXI_DeviceIPL.cpp:343 N[OSREPORT]: Instruction at 0x8015c2d4 (read from SRR0) attempted to access invalid address 0x8f34722c (read from DAR)`
Which crashes the game.
See attached log file:
[PP2.11_Crash.log](https://github.com/Birdthulu/FPM-Installer/files/4722714/PP2.11_Crash.log)

Just for fun, I tested the .elf and sd.raw in Dolphin (5.0-84d67b3-dirty) as well, and also get the same errors and eventual crashes.

My system is on Ubuntu 18.04 and using the latest Mesa (Mesa 20.0.0-devel - padoka PPA).

tl;dr Send help.

**Comment by jlambert360** [comment]
This isn't a problem with the .elf file. The elf file just launches the game. Whenever it says that an open failed like that it means it's loading it from the brawl iso. You can see here:

`05:50:384 Core/HW/EXI/EXI_DeviceIPL.cpp:343 N[OSREPORT]: File Read  (SD): Open failed (/Project+/pf/fighter/wario/FitWario11.pac)`

`05:50:611 Core/HW/EXI/EXI_DeviceIPL.cpp:343 N[OSREPORT]: File Read  (DVD): /fighter/wario/FitWario11.pac (size:715328 byte, adr:0x91b8d300) at 523`

Since it can't find the file on the sd.raw it finds it on the brawl iso and loads it from there. The game crashing when you have 2 cpus play is a separate issue with dolphin not the game. If you leave your controller idle for too long the game can crash. Hope this answers your questions!
